slider/slider.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Slider</title>
+    <link rel="stylesheet" href="./slider.css">
+
+</head>
+
+<body>
+<h1> Slider </h1>
+    <section>
+        <label for=""> Select </label>
+        <input class="slider" type="range" min="1" max="100" value="40">
```

Comment:
You can give `id` attribute to the input and then use it for the `for` attribute of label